src/types/zetkin.ts

```diff
@@ -72,16 +72,16 @@ export interface ZetkinOrganization {
 export interface ZetkinPerson {
     alt_phone: string;
     is_user: boolean;
-    zip_code: string; // Need to check
+    zip_code: string;
     last_name: string;
-    city: null; // Need to check
+    city: string;
     first_name: string;
-    gender: 'm'; // Check gender options
-    street_address: null; // Need to check
-    co_address: null; // Need to check
-    ext_id: null; // Str or num
+    gender: string;
```

Comment:
I don't know typescript well enough, but would it be possible to type this as `'f' | 'm' | 'o' | null`?

No need to fix that now. Just asking to learn, and since I seem to remember there is some sort of issue with constant types in typescript.

Reply:
Yes absolutely, another way is through an enum. I was thinking we can add these when working on the "person" views a bit more :+1: